{"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["Linux"],"languages":["swift"],"tokens":[{"text":"static","kind":"keyword"},{"text":" ","kind":"text"},{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"internalError","kind":"identifier"},{"text":": ","kind":"text"},{"text":"HTTP2ErrorCode","kind":"typeIdentifier","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/HTTP2ErrorCode","preciseIdentifier":"s:8NIOHTTP214HTTP2ErrorCodeV"}]}]}],"kind":"symbol","hierarchy":{"paths":[["doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2","doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/HTTP2ErrorCode"]]},"abstract":[{"text":"The endpoint encountered an unexpected internal error.","type":"text"}],"variants":[{"paths":["\/documentation\/niohttp2\/http2errorcode\/internalerror"],"traits":[{"interfaceLanguage":"swift"}]}],"sections":[],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/HTTP2ErrorCode\/internalError"},"metadata":{"roleHeading":"Type Property","externalID":"s:8NIOHTTP214HTTP2ErrorCodeV08internalC0ACvpZ","role":"symbol","symbolKind":"property","fragments":[{"kind":"keyword","text":"static"},{"text":" ","kind":"text"},{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"internalError","kind":"identifier"},{"text":": ","kind":"text"},{"text":"HTTP2ErrorCode","preciseIdentifier":"s:8NIOHTTP214HTTP2ErrorCodeV","kind":"typeIdentifier"}],"title":"internalError","modules":[{"name":"NIOHTTP2"}]},"schemaVersion":{"major":0,"minor":3,"patch":0},"references":{"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/HTTP2ErrorCode/internalError":{"type":"topic","role":"symbol","abstract":[{"type":"text","text":"The endpoint encountered an unexpected internal error."}],"url":"\/documentation\/niohttp2\/http2errorcode\/internalerror","kind":"symbol","fragments":[{"text":"static","kind":"keyword"},{"text":" ","kind":"text"},{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"internalError","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:8NIOHTTP214HTTP2ErrorCodeV","text":"HTTP2ErrorCode","kind":"typeIdentifier"}],"identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/HTTP2ErrorCode\/internalError","title":"internalError"},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2":{"title":"NIOHTTP2","type":"topic","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2","abstract":[],"role":"collection","kind":"symbol","url":"\/documentation\/niohttp2"},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/HTTP2ErrorCode":{"role":"symbol","kind":"symbol","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/HTTP2ErrorCode","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"HTTP2ErrorCode"}],"abstract":[{"type":"text","text":"An HTTP\/2 error code."}],"type":"topic","url":"\/documentation\/niohttp2\/http2errorcode","title":"HTTP2ErrorCode","navigatorTitle":[{"kind":"identifier","text":"HTTP2ErrorCode"}]}}}